src/org/daisy/dotify/formatter/impl/page/PageSequenceBuilder2.java

```diff
@@ -298,24 +300,47 @@ private PageImpl nextPageInner(
             boolean wasSplitInSequence,
             boolean isFirst
     ) throws PaginatorException, RestartPaginationException { // pagination must be restarted in
-        // PageStructBuilder.paginateInner
+                                                              // PageStructBuilder.paginateInner
```

Comment:
Please align this comment correctly

Reply:
What is the correct alignment?

Reply:
I would say, give it the same indent as the next line (which starts with `if`) so that the comment aligns better. Now it is very much to the right.
But this is a minor detail, don't spent too much time on it

Reply:
I've intentionally aligned it with the comment on the previous line because it's one and the same comment. It got messed up when the whole code base was reformatted.